lib/rotoscope/call_logger.rb

```diff
@@ -92,12 +92,11 @@ def log_call(call)
       return if blacklist.match?(caller_path)
       return if self == call.receiver
 
+      caller_class_name = call.caller_class_name || '<UNKNOWN>'
       if call.caller_method_name.nil?
-        caller_class_name = '<ROOT>'
```

Comment:
How is this change related?

Reply:
When tracing just starts, we won't know the caller class name and didn't know the caller method name.  However, now that we are using the ruby stack to get the caller method name, these two are orthogonal, so we can assume that just because we don't have a caller method name that we do have a class name.  As such, I needed to move this out of this conditional block.

The rename of the default value isn't as significant.  It is just a bit confusing to say that it is the root when thinking in the context of a ruby stack, so `'<UNKNOWN>'` seems like a more appropriate default value.